io.ossia.score.yml

```
@@ -0,0 +1,333 @@
+    buildsystem: simple
+    build-commands:
+      - mkdir -p "${FLATPAK_DEST}/include/"
+      - cp -rf boost "${FLATPAK_DEST}/include/"
+    sources:
+      - type: archive
+        url: https://archives.boost.io/release/1.85.0/source/boost_1_85_0.tar.bz2
+        sha256: 7009fe1faa1697476bdc7027703a2badb84e849b7b0baad5086b087b971f8617
+
+  # HAP video decoding
+  - name: snappy
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DCMAKE_POSITION_INDEPENDENT_CODE=ON
+      - -DBUILD_SHARED_LIBS=OFF
+      - -DSNAPPY_BUILD_TESTS=0
+      - -DSNAPPY_BUILD_BENCHMARKS=0
+    sources:
+      - type: archive
+        url: https://github.com/google/snappy/archive/refs/tags/1.2.1.tar.gz
+        sha256: 736aeb64d86566d2236ddffa2865ee5d7a82d26c9016b36218fcc27ea4f09f86
+
+  # ossia uses mDNS for multiple protocol discovery mechanisms
+  - name: avahi
+    builddir: true
+    config-opts:
+      - --with-distro=none
+      - --disable-qt3
+      - --disable-qt4
+      - --disable-qt5
+      - --disable-gtk
+      - --disable-gtk3
+      - --disable-glib
+      - --disable-gobject
+      - --disable-python
+      - --disable-mono
+      - --disable-monodoc
+      - --disable-libevent
+      - --disable-libdaemon
+      - --disable-dbm
+      - --disable-gdbm
+      - --disable-manpage
+
+    post-install:
+      - make -C avahi-common install
+      - make -C avahi-client install
+
+    sources:
+      - type: archive
+        url: https://github.com/avahi/avahi/releases/download/v0.8/avahi-0.8.tar.gz
+        sha256: 060309d7a333d38d951bc27598c677af1796934dbd98e1024e7ad8de798fedda
+
+  - name: ossia-score
+    buildsystem: cmake-ninja
+    builddir: true
+    no-make-install: true
+    sources:
+      - type: archive
+        url: https://github.com/ossia/score/releases/download/v3.2.3/ossia.score-3.2.3-src.tar.xz
+        sha256: 0
+    config-opts:
+      # Note: don't use RelWithDebInfo as ossia release builds explicitly expect Release build type.
+      - -DCMAKE_BUILD_TYPE=Release
+      - -Wno-dev
+      - -DSCORE_DEPLOYMENT_BUILD=1
+      - -DSCORE_STATIC_PLUGINS=1
+      - -DSCORE_FHS_BUILD=1
+      - -DCMAKE_UNITY_BUILD=1
+      - -DCMAKE_SKIP_RPATH=ON
+      - -DOSSIA_USE_SYSTEM_LIBRARIES=1
+      - -DSCORE_USE_SYSTEM_LIBRARIES=1
+      - -DSCORE_NO_QTCONF=1
+
+    post-install:
+     - cmake -DCMAKE_INSTALL_DO_STRIP=1 -DCOMPONENT=OssiaScore -P cmake_install.cmake
+     - install -Dm644 /run/build/ossia-score/cmake/Deployment/Linux/Flatpak/io.ossia.score.metainfo.xml /app/share/metainfo/io.ossia.score.metainfo.xml
+     - mv /app/share/applications/ossia-score.desktop /app/share/applications/io.ossia.score.desktop
+     - sed -i 's/Icon=ossia-score/Icon=io.ossia.score/' /app/share/applications/io.ossia.score.desktop
+     - mkdir -p /app/share/icons/hicolor/512x512/apps
+     - mv /app/share/pixmaps/ossia-score.png /app/share/icons/hicolor/512x512/apps/io.ossia.score.png
```

```suggestion
     - install -Dm644  /app/share/pixmaps/ossia-score.png -t /app/share/icons/hicolor/512x512/apps
```

And then add `rename-icon` in the top-level of the manifest. Note that you could submit a patch to upstream to install the icon in the right location. (i.e. NOT pixmaps)

---

ah, this is where appimage expects it and right now I plan to support both so..